lib/screens/home/pages/members/bloc/members_page_bloc.dart

```diff
@@ -25,47 +26,67 @@ class MembersPageBloc extends Bloc<MembersPageEvent, MembersPageState> {
     }
   }
 
-  Stream<MembersPageState> _mapEventToMembersShowed(MembersPageEvent event) async* {
+  Stream<MembersPageState> _mapEventToMembersShowed(
+      MembersPageEvent event) async* {
     final currentState = state;
 
     if (event is MembersPageShowed && !_hasReachedMax(currentState)) {
       try {
         if (currentState is MembersPageInitial) {
           yield MembersPageLoading();
-          final List<User> users = await userRepository.getVerifiedUsers(pageNumber);
-          yield MembersPageSuccess(users: users, hasReachedMax: false);
+          final List<User> users =
+              await userRepository.getVerifiedUsers(pageNumber);
+          final User currentUser = await userRepository.getCurrentUser();
```

Comment:
This network call will be made when the state is _MembersPageInitial_. This is a necessary call.